test/test_torch.py

```diff
@@ -3094,59 +3094,54 @@ def _test_broadcast(self, cast):
                 # map and map2 are not implementd on CUDA tensors
                 continue
 
-            # TODO: fix masked_scatter and masked_fill broadcasting
-            if hasattr(large_expanded, fn) and fn not in ['masked_scatter', 'masked_fill']:
```

Comment:
@ezyang this change is because `['masked_scatter', 'masked_fill']` does not support broadcasting on the first argument, so it is excluded from the test, now I need to add them back.